### DISCUSSION TITLE
problem with kitty -single-instance

### DISCUSSION THREAD

**Discussion by yaroslavyaroslav** [kovidgoyal/kitty#7788] [Q&A] [answered]
I'm trying to run kitty as a single instance on my mac, but every time I call `kitty -1` I got `Failed to connect to single instance socket: Connection refused` no matter whether i'm calling it with no kitty process already running or not. Anyone met this before? Any ideas to fix this?

**Comment by kovidgoyal** [comment, marked as answer]
That means the socket already exists. Presumably a stale socket file left over from a crash. Reboot your computer and you should be fine. Or manually delete the socket which on macOS you can find with something like

```sh
find /private -name 'kitty-ipc*' 2>/dev/null
```

**Reply by yaroslavyaroslav** [reply]
Yeah, thanks, that helped.
